react/features/connection-indicator/components/web/BandwidthSettingsDialog.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { KeyboardEvent, useCallback, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import { makeStyles } from 'tss-react/mui';
+
+import { IReduxState } from '../../../app/types';
+import { setAssumedBandwidthBps as saveAssumedBandwidthBps } from '../../../base/conference/actions';
+import { IconInfoCircle } from '../../../base/icons/svg';
+import { withPixelLineHeight } from '../../../base/styles/functions.web';
+import Dialog from '../../../base/ui/components/web/Dialog';
+import Input from '../../../base/ui/components/web/Input';
+
+const useStyles = makeStyles()(theme => {
+    return {
+        content: {
+            color: theme.palette.text01
+        },
+
+        info: {
+            background: theme.palette.ui01,
+            ...withPixelLineHeight(theme.typography.labelRegular),
+            color: theme.palette.text02,
+            marginTop: theme.spacing(2)
+        },
+
+        possibleValues: {
+            margin: 0,
+            paddingLeft: theme.spacing(4)
+        }
+    };
+});
+
+/**
+ * Bandwidth settings dialog component.
+ *
+ * @returns {ReactElement}
+ */
+const BandwidthSettingsDialog = () => {
+    const { classes } = useStyles();
+    const { t } = useTranslation();
+    const dispatch = useDispatch();
+    const [ showAssumedBandwidthInfo, setShowAssumedBandwidthInfo ] = useState(false);
+    const currentAssumedBandwidthBps = useSelector(
+        (state: IReduxState) => state['features/base/conference'].assumedBandwidthBps
+    );
+    const [ assumedBandwidthBps, setAssumedBandwidthBps ] = useState(
+        currentAssumedBandwidthBps === -1
```

Comment:
Should we use the constant here?

Reply:
That's right, I refactored it after quite some time and forgot about the constant